WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -772,7 +799,6 @@ - (void)setEditing:(BOOL)editing animated:(BOOL)animated
     [super setEditing:editing animated:animated];
     [self.tableView setEditing:editing animated:animated];
     self.dataSource.editing = editing;
-    [self toggleRightBarButtonItems:!editing];
```

Comment:
With this change the `+` button no longer toggles/disables in edit mode. I think that functionality should remain the same.

Reply:
Hrm. I thought the end functionality was the same. Maybe I missed what this actually does. I'll take another look.